zproject/settings.py

```diff
@@ -1079,3 +1079,5 @@ def get_secret(key):
 PROFILE_ALL_REQUESTS = False
 
 CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+
+USE_SOCKET = True
```

Comment:
This should be added to DEFAULT_SETTINGs instead of here; see http://zulip.readthedocs.io/en/latest/settings.html for an explanation as to why.

Also, can you call it `USE_WEBSOCKETS`?  I think that name is clearer :)